### PR TITLE
Task2

### PR DESCRIPTION
**Pull request by alexandraPatica**

### CODE REVIEW

File: Patica Andreea/FixMe.c [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+void transpose(int **a,int n,int m)
+{
+    int i,j;
+    int** b=(int**)(malloc(sizeof(int)*m));
+    for(i=0; i<m; i++)
+        b[i]=(int*)(malloc(sizeof(int)*n));
+    for(i=0; i<n; i++)
+        for(j=0; j<m; j++)
+            b[j][i]=a[i][j];
+    print(b,m,n);
```
Comment (Alex-Cosma) [inline Patica Andreea/FixMe.c:26]: I don't like this. You say the function `transposes` something, yet it doesn't.
It `PRINTS` the matrix transposition, but doesn't return it, doesn't change the matrix I've sent, it's very, very misleading.
You might not agree with me now, but you will soon enough. This function should either change the elements "in place" or return the transposed matrix.

File: Patica Andreea/Lab11/Task2/main.c [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+        {
+            number=number+(sourceString[i]-'0')*p;
+            p/=10;
+        }
+    }
+//    printf("%f\n", number);
+    return number;
+}
+
+void function(char *strbuf, items *item)
```
Comment (Alex-Cosma) [inline Patica Andreea/Lab11/Task2/main.c]: the name of your function is `function`!? :))
What in the world should it do?

**Review by Alex-Cosma — CHANGES_REQUESTED** [review]
Please work on your code quality and submit things afterwards. Thank you!

File: Patica Andreea/FixMe.c (hunk 1 above)
Comment (alexandraPatica) [reply Patica Andreea/FixMe.c:26]: I did not write it!. This is the code in which you told us to correct the errors

Comment (Alex-Cosma) [reply Patica Andreea/FixMe.c:26]: Woops

**Comment by Alex-Cosma** [comment]
👍

**Pull request merged**